style.css

```diff
@@ -286,3 +286,114 @@ h5 {
   font-weight: 500;
   line-height: 24px;
 }
+
+/* for contact form */
+form {
+  background-color: #191765;
+  width: 99.9%;
+  margin: 0;
+  text-align: center;
+}
+
+.form-paragraph {
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  padding: 10%;
+}
```

Comment:
- [ ] Kindly make sure the `form-paragraph` text uses the correct color and spacing(top & bottom) like the one from the Figma template to make your project sticks to the Figma design.